### PR TITLE
feat(tier4_perception_launch): add option for euclidean lidar detection model

### PR DESCRIPTION
**Pull request by shmpwk**
## Description

Add euclidean lidar deteciton model option instead of apollo or centerpoint model.
This option is useful for no-cuda users since apollo and centerpoint model need cuda.

## Related links

[TIER IV internal link](https://github.com/tier4/autoware_launch.s1/pull/158)

## Tests performed
- I tested this option with logiee and confirmed it works.

## Notes for reviewers
- I changed the option only for [lidar_based_detection.launch.xml](https://github.com/team-tier4-FY22/autoware.universe.fy22/blob/main/launch/tier4_perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml) but not for [camera_lidar_fusion_based_detection.launch.xml](https://github.com/team-tier4-FY22/autoware.universe.fy22/blob/main/launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml) under [detection.launch.xml](https://github.com/team-tier4-FY22/autoware.universe.fy22/blob/main/launch/tier4_perception_launch/launch/object_recognition/detection/detection.launch.xml).
This is because  [camera_lidar_fusion_based_detection.launch.xml](https://github.com/team-tier4-FY22/autoware.universe.fy22/blob/main/launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml) can detect objects [here](https://github.com/team-tier4-FY22/autoware.universe.fy22/blob/main/launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml#L154-L157) without apollo or centerpoint model by default.

- I only check this option with logiee, so please be sure to check the other reference design.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by taikitanaka3** [comment]
@shmpwk 
Can you make this PR to tier4 fork autoware_launch too?

**Comment by yukkysaito** [comment]
I'm currently changing centerpoint and apollo with the clustering outputs, so can you please wait for that? I will send out a PR soon.

**Comment by shmpwk** [comment]
Thank you for checking, I will change this PR to draft until centerpoint and apollo with the clustering outputs PR.

**Comment by shmpwk** [comment]
@yukkysaito 
> I'm currently changing centerpoint and apollo with the clustering outputs, so can you please wait for that? I will send out a PR soon.

Have you finished it yet?

**Comment by yukkysaito** [comment]
@shmpwk Sorry for late. I finished. :pray:

**Comment by shmpwk** [comment]
I confirmed it works with AWSIM by following command.
```
ros2 launch autoware_launch e2e_simulator.launch.xml vehicle_model:=sample_vehicle sensor_model:=awsim_sensor_kit map_path:=/home/shumpeiwakabayashi/Downloads/nishishinjuku_autoware_map/ lidar_detection_model:=clustering
```

Screenshot from 2022-12-23 11-35-53

**Comment by shmpwk** [comment]
I will edit autoware_launch too.

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
   <arg name="image_raw7" default="/image_raw7"/>
   <arg name="camera_info7" default="/camera_info7"/>
   <arg name="image_number" default="1" description="choose image raw number(0-7)"/>
-  <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`"/>
+  <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`, `clustering`"/>
```
Comment (shmpwk) [inline launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:23]: @miursh 
Can I add description of lidar_detection_model for pointpainting?

Comment (yukke42) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:23]: @shmpwk No, It's only for camera-lidar-fusion.

**Comment by yukke42** [comment]
@shmpwk Cloud you check launch files again?  The options are only changed.

**Comment by shmpwk** [comment]
Actually, there is no need to add some script.

**Comment by shmpwk** [comment]
Resolved conflict. Any other concerns for this PR?

**Review by miursh — APPROVED** [review]
LGTM

**Pull request merged**